### ISSUE TITLE
Rendering bugs when using auidemo

### ISSUE THREAD

**Issue by robertop** [nhold/wxWidgets#74]
I compiled version  601023671139d269c26d59034062530d0af88ba8 on  Linux 64-bit, GTK2 toolkit
1. Run the auidemo
2. maximize the window
3. Go to View -> Create wxAuiNotebook
4. Click the "x" to close the newly created notebook
5. Go to Dynamic Notebooks -> Toggle Dynamic Notebook Style
6. There is a rendering error on the screen (left side). See screenshot.
   ![auidemo-bug](https://cloud.githubusercontent.com/assets/1117179/4782072/1d1c9f38-5cd9-11e4-9c20-068d817014f4.png)
7. Restore the window to normal size by double-clicking on the main frame's title bar
8. Rendering errors are even more pronounced. See screenshot.
   ![auidemo-bug-2](https://cloud.githubusercontent.com/assets/1117179/4782074/2da533d8-5cd9-11e4-82cd-30588cbf7ad5.png)

**Comment by mjmacleod** [comment]
Hi Roberto,

I can't reproduce that on my linux machine. What distro and window manager?
Can you confirm that the normal AUI in main wxWidgets branch doesn't have similar problems?

Thanks,
Malcolm

**Comment by robertop** [comment]
I am using ubuntu 12.04, 64 bit.  Did some testing with wxWidgets trunk and this same issue happens there, so I don't think this is a problem isolated to this branch.

EDIT: using the default desktop manager (ubuntu unity). I am pretty sure its not just my machine, since I have been able to reproduce it on clean VMs.

**Comment by mjmacleod** [comment]
Okay thanks,
I suspect then that it is a unity issue...
While obviously someone needs to look into it, as you say it happens also in trunk so I strongly suggest we leave this for after the merge, or somebody looks at it on trunk in the meantime, rather than holding the merge up over a trunk issue.

**Comment by dronesalot** [comment]
![aui-bugs](https://cloud.githubusercontent.com/assets/9636311/5597657/2e8f3688-9274-11e4-9e19-f7fd6175a4e4.jpg)

The current demo in MSW also has some odd display characteristics. It seems to be related to the bitmap sizing.

**Comment by robertop** [comment]
The rendering issues in Ubuntu are due to this issue http://trac.wxwidgets.org/ticket/16795. 

TL;DR: disabling ubuntu's overlay scrollbars makes the rendering issues disappear.

**Comment by mjmacleod** [comment]
This should I believe be a 'future' milestone, as it is not specifically to do with this branch.

**Comment by mjmacleod** [comment]
formerge milestone should be deleted I think?

**Comment by Kinaou** [comment]
As @robertop said:

> The rendering issues in Ubuntu are due to this issue http://trac.wxwidgets.org/ticket/16795.

This issue being closed, is it mean the problem is solved?

Perhaps it could be nice to update the aui-dynamic-notebook branch with the latest master trunk state?

**Comment by nhold** [comment]
@Kinaou Yeah, that is currently non-trival. If you want to take a crack at it you can, I have been trying to find the time to do so for a while now.
